gallery/src/components/NxForm/NxFormCustomizedExample.scss

```diff
@@ -4,9 +4,16 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
+@use '~@sonatype/react-shared-components/scss-shared/nx-dark-mode-helpers';
+
 .gallery-custom-form {
   background-color: #eee;
   padding: var(--nx-spacing-4x);
+
+  // use dark mode helper here as it is for demonstration purpose only
```

Comment:
Keep in mind that this code is part of the example, readable by the gallery user.  What is this comment intended to convey to that audience?

Reply:
Since we are creating specific variables for the color used in component in dark mode, leave the comments here that this code is for example purpose and no need to create a new variable.